test/tf/k8s/main.tf

```diff
@@ -53,6 +53,7 @@ users:
   user:
     username: ${google_container_cluster.cluster.master_auth[0].username}
     password: ${google_container_cluster.cluster.master_auth[0].password}
+
```

Comment:
Without this change I was getting this error (https://circleci.com/gh/hashicorp/go-discover/1364):

```
    k8s_discover_test.go:41: 2 errors occurred:
        	* discover-k8s: error loading kubeconfig: error loading config file "/home/circleci/project/test/tf/k8s/kubeconfig.yaml": yaml: line 19: did not find expected whitespace or line break
        	* discover-k8s: error loading in-cluster config: unable to load in-cluster configuration, KUBERNETES_SERVICE_HOST and KUBERNETES_SERVICE_PORT must be defined
```